### PR TITLE
[tests] assign sig labels to the unlabelled tests

### PR DESCRIPTION
**Pull request by enp0s3**
Signed-off-by: Igor Bezukh <ibezukh@redhat.com>

**What this PR does / why we need it**:
For a proper distribution of tests among the CI lanes

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Review by fgimenez — APPROVED** [review]
/lgtm

Thanks!

**Comment by fgimenez** [comment]
Nice, no tests remaining in the 1.19 lane with the changes in this PR https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/pr-logs/pull/kubevirt_kubevirt/5741/pull-kubevirt-e2e-k8s-1.19/1399347320193028096 :+1:

File: tests/events_test.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ const (
 	diskName   = "disk0"
 )
 
-var _ = Describe("[Serial] K8s IO events", func() {
+var _ = Describe("[Serial][sig-storage]K8s IO events", func() {
```
Comment (EdDev) [inline tests/events_test.go:29]: This one is very strange. It checks "events" but through storage entities.

If it really belongs to storage, it probably should move to the `storage` folder (although not a must in this PR).

**Review by EdDev — COMMENTED** [review]
Except the inline comment, a general note: It will be nice to move the compute tests to its own folder as the storage and network sigs have done.

File: tests/events_test.go (hunk 1 above)
Comment (dhiller) [reply tests/events_test.go:29]: @mhenriks can you confirm that? Thanks!

**Review by dhiller — APPROVED** [review]
/approve
/hold

Waiting for confirmation, after that is resolved, feel free to unhold.

File: tests/events_test.go (hunk 1 above)
Comment (dhiller) [reply tests/events_test.go:29]: @awels can you have a look here?

Comment (awels) [reply tests/events_test.go:29]: Yes it does look like it should belong in storage.

**Pull request merged**